package.json

```diff
@@ -117,22 +117,22 @@
     "cross-env": "^6.0.3",
     "css-loader": "^1.0.0",
     "cypress": "^8.3.1",
-    "eslint": "^7.7.0",
+    "eslint": "^8.57.1",
```

Comment:
the version of eslint we were using similarly had a peer dependency on typescript <v4

v9 has some breaking changes with eslint config, again, something we should address, but to keep the change surface low, I've gone with v8